.github/workflows/tests.yaml

```diff
@@ -22,9 +22,13 @@ jobs:
       matrix:
         include:
           - php-version: "8.1"
-            dependencies: "lowest"
+            dependencies: "lowest" # Pimcore 10.5.0
           - php-version: "8.1"
-            dependencies: "highest"
+            dependencies: "highest" # Pimcore 11.*
+          - php-version: "8.2"
+            dependencies: "lowest" # Pimcore 10.5.0
```

Comment:
Note that this is not officially supported.

Reply:
Jep. But i wanted to change as least as possible :)